tests/core/object/test_class_db.h

```diff
@@ -378,6 +378,9 @@ void validate_argument(const Context &p_context, const ExposedClass &p_class, co
 #ifdef DEBUG_METHODS_ENABLED
 	TEST_COND((p_arg.name.is_empty() || p_arg.name.begins_with("_unnamed_arg")),
 			vformat("Unnamed argument in position %d of %s '%s.%s'.", p_arg.position, p_owner_type, p_class.name, p_owner_name));
+
+	TEST_FAIL_COND((p_arg.name != "@varargs@" && !p_arg.name.is_valid_identifier()),
+			vformat("Invalid argument name %s of %s '%s.%s'.", p_arg.name, p_owner_type, p_class.name, p_owner_name));
```

Comment:
```suggestion
			vformat("Invalid argument name '%s' of %s '%s.%s'.", p_arg.name, p_owner_type, p_class.name, p_owner_name));
```
Will add for clarity